src/pgrn-groonga.c

```diff
@@ -710,3 +710,44 @@ PGrnExprAppendConstString(grn_obj *expr,
 			  string,
 			  nArgs);
 }
+
+void
+PGrnExprAppendObject(grn_obj *expr,
+					 grn_obj *object,
+					 grn_operator op,
+					 int nArgs,
+					 const char *tag,
+					 const char *format,
+					 ...)
+{
+	grn_expr_append_obj(ctx, expr, object, op, nArgs);
+	if (ctx->rc == GRN_SUCCESS)
+		return;
+
+	if (format)
+	{
+#define MESSAGE_SIZE 4096
+		va_list args;
+		char message[MESSAGE_SIZE];
+
+		va_start(args, format);
+		grn_vsnprintf(message, MESSAGE_SIZE, format, args);
+		va_end(args);
+
+		PGrnCheck("%s: failed to %s object: %s(%d): %s",
```

Comment:
あぁ、`nArgs`は`op`の後につけようか。`object`よりは`op`に関係しているんよ。

```suggestion
		PGrnCheck("%s: failed to %s(%d) object: %s: %s",
```